src/examples/v2/mina-program.ts

```diff
@@ -0,0 +1,127 @@
+import { Bool, Experimental, Field, PrivateKey, PublicKey, UInt32 } from 'o1js';
+
+const V2 = Experimental.V2;
+
+type TestState = {
+  x: typeof Field;
+  address: typeof PublicKey;
+};
+
+const TestState: Experimental.V2.StateDefinition<TestState> = V2.State({
+  x: Field,
+  address: PublicKey,
+});
+
+const TestProgram = V2.MinaProgram({
+  name: 'TestProgram',
+  State: TestState,
+  Event: V2.GenericData,
+  Action: V2.GenericData,
+
+  methods: {
+    init: {
+      privateInputs: [Field],
+
+      async method(
+        env: Experimental.V2.MinaProgramEnv<TestState>,
+        value: Field
+      ): Promise<Experimental.V2.MinaProgramMethodReturn<TestState>> {
+        return {
+          incrementNonce: new Bool(true),
+          preconditions: {
+            account: {
+              isProven: new Bool(false),
+              nonce: new UInt32(0),
+            },
+          },
+
+          setState: {
+            x: V2.Update.from(value, Field(0)) as never,
+            address: V2.Update.from(env.accountId.publicKey, PublicKey.empty()) as never,
```

Comment:
Only the `x` value needs the casting as never.  Do you know why the typing for setState doesn't allow an update for x?